pkg/server/authentication.go

```diff
@@ -22,10 +22,12 @@ import (
 	"encoding/base64"
 	"fmt"
 	"net/http"
+	"strconv"
 	"time"
 
 	gwruntime "github.com/grpc-ecosystem/grpc-gateway/runtime"
 	"github.com/pkg/errors"
+	context2 "golang.org/x/net/context"
```

Comment:
The version of grpc-gateway we use requires this, but it makes the linter fail. Newer version uses stdlib `context`; will bump.

Reply:
tried this: ```diff
diff --git a/Gopkg.toml b/Gopkg.toml
index c173db9a5..dfc4526f9 100644
--- a/Gopkg.toml
+++ b/Gopkg.toml
@@ -71,6 +71,11 @@ ignored = [
   name = "cloud.google.com/go"
   revision = "094187a3a68a589c8217a46b90af0efae38542b9"
 
+# pull in https://github.com/grpc-ecosystem/grpc-gateway/pull/596
+[[constraint]]
+  name = "github.com/grpc-ecosystem/grpc-gateway"
+  revision = "37660b1a4ff83f0ac3672ff7976223a859859e2f"
+
 # github.com/docker/docker depends on a few functions not included in the
 # latest release: reference.{FamiliarName,ParseNormalizedNamed,TagNameOnly}.
 #
```

But running `godep ensure` created a huge diff in `Gopkg.lock` and `vendor/`. @dt is that ok, or is there a way to make a more minimal dep change?